notebooks/14-combine-data.md

```diff
@@ -138,11 +138,13 @@ def add_date_dimension(ds):
 
 ```{code-cell} ipython3
 moisture_index_lazy = xr.open_mfdataset(sorted(Path("./data/herstappe/raster/sentinel_moisture").rglob("*.tiff")), 
-                                        preprocess=add_date_dimension, engine="rasterio", decode_cf=False) # parallel=True
+                                        preprocess=add_date_dimension, engine="rasterio", decode_cf=True, 
```

Comment:
Is there a specific reason you changed the `decode_cf=True`? (I assume we can actually also just leave this out, as I don't think the rasterio engine handles that; this is typically for netcdf like files)

Reply:
Ah, with `False` it gives an error now ..